### ISSUE TITLE
Can't override properties in trait

### ISSUE THREAD

**Issue by paras-malhotra** [elasticquent/Elasticquent#42]
From the PHP documentation: 

> If a trait defines a property then a class can not define a property with the same name, otherwise an error is issued. It is an E_STRICT if the class definition is compatible (same visibility and initial value) or fatal error otherwise.

The properties of the [Elasticquent trait](https://github.com/elasticquent/Elasticquent/blob/master/src/ElasticquentTrait.php) (such as mappingProperties and usesTimestampsInIndex) right now cannot be overriden by directly using the trait in the model class. We would need to make a base model class use the ElastquentTrait and then override the properties.

I would suggest something like so:

```
public function usesTimestampsInIndex() {
    return property_exists($this, 'usesTimestampsInIndex') ? $this->usesTimestampsInIndex : true;
}
```

and possibly removing the property declarations in the trait to be able to directly use the trait in the model class.

Just my 2 cents!

**Comment by onbjerg** [comment]
Hi @timgws. This seems to be fixed as of 97a4d87?

**Comment by timgws** [comment]
Correct.
